src/app/(auth)/sign-in/page.tsx

```diff
@@ -41,35 +45,59 @@ function SingIn() {
   };
 
   return (
-    <div>
-      <Formik
-        initialValues={iniatialValues}
-        onSubmit={(values) => signInUser(values)}
+    <div className="grid grid-cols-1 md:grid-cols-2 h-screen">
+      <div
+        className="hidden lg:block bg-cover bg-center"
+        style={{ backgroundImage: "url('/images/login-bg.png')" }}
       >
-        {(props) => (
-          <Form>
-            <TextField
-              type="email"
-              name="email"
-              label="Email"
-              placeholder="loremipsum@gmail.com"
-            />
-            <TextField
-              type="password"
-              name="password"
-              label="Password"
-              placeholder="qwert@123"
-            />
-            <button type="submit">Log in</button>
-          </Form>
-        )}
-      </Formik>
-      {isSeller && (
-        <div>
-          logged in
-          <button onClick={() => logOut()}>log out</button>
+        <div className="flex flex-col h-full space-y-0">
+          <div className="bg-white h-[128px] py-[32px] pl-[64px]">
+            <text className="text-center text-[#EC1C24] text-4xl font-bold mt-[40px] mb-[40px]">
+              Selise Pan Dokan
+            </text>
+          </div>
+
+          <section className="grid grid-cols-4 grid-rows-2 mb-[80px]">
+            <div className="bg-[#FDDFD9] h-[189px]"></div>
+            <div className="bg-[#FFFFFF] h-[189px]"></div>
+            <div className="bg-[#F5BBAA] h-[189px]"></div>
+            <div className="bg-[#FDDFD9] h-[189px]"></div>
+            <div className="bg-[#FFFFFF] h-[189px]"></div>
+            <div className="bg-[#FDDFD9] h-[189px]"></div>
+            <div className="bg-[#FF6F61] h-[189px]"></div>
+            <div className="bg-[#FCF1E8] h-[189px]"></div>
+          </section>
+
+          <section className="text-left mb-[40px] pt-[32px] pl-[64px] space-y-8">
+            <h2 className="text-xl text-white font-semibold m">Our Mission</h2>
```

Comment:
Please refrain from using tailwind classes directly esp if it is unit related. We will have px unit for all space, margin, padding, and fonttsize

Reply:
Please check all the files again